pythonx/jedi_vim.py

```diff
@@ -193,6 +193,13 @@ def get_environment(use_cache=True):
     return environment
 
 
+def get_known_environments():
+    """Get known Jedi environments."""
+    envs = list(jedi.api.environment.find_virtualenvs(paths=[], safe=False))
```

Comment:
Why overwrite safe? It's not like people are really using that (nor will they probably ever in jedi-vim) and also why paths?

Reply:
`paths` is used in #836 - thought it would be easier to merge then probably by keping it here.
´safe=False` is used because the user configures `paths` in #836.
I will change it here to not use any args.